### DISCUSSION TITLE
meta-freescale or meta-imx?

### DISCUSSION THREAD

**Discussion by macaquinyho** [Freescale/meta-freescale#1842] [General]
Sorry if this is not the appropiate place for asking this kind of question, but I've wondering what are the advantages or downsides of using meta-freescale compared to meta-imx for a recent product (imx93). Right now it seems it uses linux-imx for imx93, so it should be the same as meta-imx.

I noticed meta-imx is full of workarounds and bad practises (overriding qtbase for imx93 removing linuxfb??)https://github.com/nxp-imx/meta-imx/blob/773c0049c3b35af8f5b5bdf8da78a6236a4fc184/meta-imx-sdk/conf/distro/include/fsl-imx-base.inc#L47.

**Comment by fabioestevam** [comment]
meta-imx is still using nanbield which is EOL.

With meta-freescale you can use scarthgap with imx93-evk.

**Comment by angolini** [comment]
I converted the issue to discussion, as this is a common question, and I would not like to have the answer buried in a closed issue.

As soon as a new BSP release is available in `meta-imx`, we all (community and nxp) work together to upstream the BSP to `meta-freescale` master.

So, the goal is to have the same latest BSP available on a fixed meta layer (meta-imx) and on the latest yocto project base (meta-freescale)

In addition, on `meta-freescale`, you can choose (when available) between mainline or NXP BSP.
